### ISSUE TITLE
Implement sem status operation to debug dead-lock cases

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#2145]
To implement a new operations (e.g. `/sem`) which would return 

```
{
    "request" : { "status": "open", "time: 0" },
    "request" : { "status": "closed", "time: 0.002" },
    "transaction" : ...
    "subCache" : ...
    "connectionContext" : ...
    "timeStat" : ...
  }
}
```

If providing blocking times is complicated, then just open/close information for each sem would be ok.

**Comment by kzangeli** [comment]
Only status in the list of sem-ifnfo so far.

In the future, more info should be added:

*  **owner**, in case 'taken': who took it?
* **purpose**, in case 'taken': for what purpose?
* **errors**, number of errors for this semaphore
* **waitTime**, total waiting-time for this semaphore
* **count**, number of times the semaphore has been taken

**Comment by fgalan** [comment]
> waitTime, total waiting-time for this semaphore

Not really... waitTime on sem is already in /statitics ops. What would be interesting in /sem would be a metric (named waitTime or other) that:

* If the sem is "free" is always "0"
* If the same is "taken" is the time that it has been taken.

This is useful to check if a sem is never released (i.e. if we always see it in "taken" state and the time metric dont't stop to grow).

**Comment by fgalan** [comment]
Partically implemented in PR https://github.com/telefonicaid/fiware-orion/pull/2234.

I understand that with that implementation target for 1.3.0 is covered. Thus, moving out of milestone.
